<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BiswaranjanD-PC" name="Command line test 559227fc-c462-48ab-bdde-001d5dce266c" tests="2" failures="1" timestamp="21 Jul 2017 12:13:52 GMT" time="108.639" errors="0">
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.loylty.mobileautomation.testcases.Bom_Test">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.loylty.mobileautomation.testcases.Bom_Test.getResult(Bom_Test.java:124)
	at com.loylty.automationframework.TestRunner.main(TestRunner.java:121)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@BeforeMethod setup" time="0.0" classname="com.loylty.mobileautomation.testcases.Bom_Test">
    <skipped/>
  </testcase> <!-- @BeforeMethod setup -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.loylty.mobileautomation.testcases.Bom_Test">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="recharge" time="93.893" classname="com.loylty.mobileautomation.testcases.Bom_Test">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.loylty.mobileautomation.testcases.Bom_Test.recharge(Bom_Test.java:103)
	at com.loylty.automationframework.TestRunner.main(TestRunner.java:121)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- recharge -->
  <testcase name="verify_humberger" time="0.0" classname="com.loylty.mobileautomation.testcases.Bom_Test">
    <skipped/>
  </testcase> <!-- verify_humberger -->
</testsuite> <!-- Command line test 559227fc-c462-48ab-bdde-001d5dce266c -->
